backend/config/.env.example

```
@@ -59,6 +59,10 @@ AWS_SNS_TOPIC_ARN=arn:aws:sns:eu-north-1:123456789012:owear
 
 #--- SYNC SETTINGS ---#
 SYNC_INTERVAL_SECONDS=3600  # How often to run automatic sync (default: 1 hour)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)